packages/web/src/components/range/MultiRange.js

```diff
@@ -240,29 +240,43 @@ class MultiRange extends Component {
 						{this.props.title}
 					</Title>
 				)}
-				<UL className={getClassName(this.props.innerClass, 'list') || null}>
-					{this.props.data.map(item => (
-						<li
-							key={item.label}
-							className={`${this.state.selectedValues[item.label] ? 'active' : ''}`}
-						>
-							<Checkbox
-								className={getClassName(this.props.innerClass, 'checkbox') || null}
-								id={`${this.props.componentId}-${item.label}`}
-								name={this.props.componentId}
-								value={item.label}
-								onChange={this.handleClick}
-								checked={!!this.state.selectedValues[item.label]}
-								show={this.props.showCheckbox}
-							/>
-							<label
-								className={getClassName(this.props.innerClass, 'label') || null}
-								htmlFor={`${this.props.componentId}-${item.label}`}
+				<UL
+					className={getClassName(this.props.innerClass, 'list') || null}
+					role="listbox"
+					aria-label={`${this.props.componentId}-items`}
+				>
+					{this.props.data.map((item) => {
+						const isChecked = !!this.state.selectedValues[item.label];
+						return (
+							<li
+								key={item.label}
+								className={`${
+									this.state.selectedValues[item.label] ? 'active' : ''
+								}`}
```

Comment:
@jyash97 What's the point of adding a variable if you're not using it.

Reply:
Missed this line will fix it